samples/cpp/facedetect.cpp

```diff
@@ -202,18 +183,21 @@ void detectAndDraw( Mat& img, CascadeClassifier& cascade,
     int i = 0;
     double t = 0;
     vector<Rect> faces, faces2;
-    const static Scalar colors[] =  { CV_RGB(0,0,255),
-        CV_RGB(0,128,255),
-        CV_RGB(0,255,255),
-        CV_RGB(0,255,0),
-        CV_RGB(255,128,0),
-        CV_RGB(255,255,0),
-        CV_RGB(255,0,0),
-        CV_RGB(255,0,255)} ;
-    Mat gray, smallImg( cvRound (img.rows/scale), cvRound(img.cols/scale), CV_8UC1 );
+    const static Scalar colors[] =
+    {
+        Scalar(0,0,255),
```

Comment:
Attention here, previous values where RGB, but the Scalars handle BGR values... So you should switch indexes around to maintain consistency! Goes for all the following scalar values.